internal/autotag/tag_test.go

```diff
@@ -1,6 +1,7 @@
 package autotag
```

Comment:
Only related to this method definition change: https://github.com/stashapp/stash/pull/4423/files?diff=split&w=1#r1443579818
No change in behavior expected.